dash_docs/chapters/dash_vtk/index.py

```diff
@@ -0,0 +1,865 @@
+    - __VolumeRepresentation__: The volume representation will expect a 3D image and will render it using a Volume Rendering technique that will let you see through (foggy object).
+    - __SliceRepresentation__: The slice representation will expect a 3D image and will slice it along a given axis.
+
+    Representations should be put inside the children of a __View__.
+
+    ### DataSource
+
+    A __DataSource__ can be many things but it is mostly something that can produce data. In other words it could be a `dataset` or a `filter` that consume some data and generate new ones or even a `reader` that will read somekind of input (file, url...) and produce some data. Any __DataSource__ can be placed inside the children of another __DataSource__ that will act as a filter or simply passed to a __Representation__.
+
+    In `dash_vtk` we have several objects that falls into that category. The list below gives you an overview of those but more details information can be found later.
+    - __Algorithm__: It allow you to instanciate a vtk.js algorithm that could either be a filter (vtkWarpScalar) or a source (vtkLineSource, vtkConeSource, vtkPlaneSource, vtkSphereSource, vtkCylinderSource).
+    - __ImageData__: An ImageData is what I've been calling a 3D image so far. This element will let you define each piece that compose a 3D image.
+    - __PolyData__: A PolyData is a surface mesh (points, triangles...). This element will let you define the various piece of a mesh.
+    - __Reader__: A reader is similar to an __Algorithm__ except that readers have a common set of API and this element let you leverage those.
+    - __ShareDataSet__: This element allow you to capture any __DataSource__ and make it available into another processing pipeline or representation without duplicating the data that get sent from the server to the client.
+    - __Mesh__: This element is similar to __PolyData__ except that it has a Python helper function that goes with it which will help you map a __vtkDataSet__ into a single property of the __Mesh__.
+    - __Volume__: This element is similar to __ImageData__ except that it has a Python helper function that goes with it which will help you map a __vtkImageData__ into a single property of the __Volume__.
+
+    ## Geometry Rendering
+
+    Now that we have those core concepts down we can show you some code on how to render a mesh using `dash-vtk`.
+
+    '''),
+
+    html.Details(open=False, children=[
+        html.Summary('View full code'),      
+        rc.Markdown(
+            examples['t00_geometry_rendering.py'][0], 
+            style=styles.code_container
+        ),
+    ]),
+
+    html.Div(
+        examples['t00_geometry_rendering.py'][1], 
+        className='example-container'
+    ),
+
+    rc.Markdown('''
+    ```py
+    # Use helper to get a mesh structure that can be passed as-is to a Mesh
+    mesh_state = to_mesh_state(dataset)
```

Comment:
```suggestion
    mesh_state = dash_vtk.utils.to_mesh_state(dataset)
```
unless you want to show its import statement

Reply:
The code snippets explicitly import it so I guess we could be consistent in that way:

```suggestion
    from dash_vtk.utils import to_mesh_state
    mesh_state = to_mesh_state(dataset)
```

Reply:
yep, that's what I meant by "unless you want to show its import statement" - whichever you prefer is fine, just want to make sure the code stands on its own.